save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserInfo.kt

```diff
@@ -55,6 +56,7 @@ data class UserInfo(
     val freeText: String? = null,
     val realName: String? = null,
     val createDate: LocalDateTime? = null,
+    val userPermissions: UserPermissions = UserPermissions.empty,
```

Comment:
having such optional fields complicates the logic -- `UserInfo` doesn't guarantee that it has `userPermissions`, because it's required using some another endpoint.

Can be keep `UserInfo` is consistent? If we need `userPermissions`, we can make an another request to backend or create a new data class with extended info about user

Reply:
Ok, agree. Removed it